deploy/csi-operator.yaml

```diff
@@ -0,0 +1,1026 @@
+# This manifest deploys the OpenEBS CSI control plane components,
```

Comment:
Can we make use of this operator.yaml to install CSI in `Travis` instead of other [YAML](https://github.com/openebs/cstor-operators/blob/82435a2bd6d46fe1a145ba4d7bc2324d4d939100/ci/sanity/install.sh#L37)(It will give more confidence)

Reply:
Yes, i was doing that change in Policy docs PR , but let me change it here itself

Reply:
done